packages/cloud-cognitive/src/components/ActionBar/ActionBarItem.js

```diff
@@ -5,18 +5,27 @@
 // LICENSE file in the root directory of this source tree.
 //
 
+// Import portions of React that are needed.
 import React from 'react';
-import PropTypes from 'prop-types';
 
+// Other standard imports.
+import PropTypes from 'prop-types';
 import cx from 'classnames';
+import { pkg } from '../../settings';
 
+// Carbon and package components we use.
 import { Button } from 'carbon-components-react';
 
-import { pkg } from '../../settings';
+// The block part of our conventional BEM class names (blockClass__E--M).
 const componentName = 'ActionBarItem';
 const blockClass = `${pkg.prefix}-action-bar-item`;
 
-export let ActionBarItem = ({ inOverflow, ...otherProps }) => {
+// NOTE: the component SCSS is not imported here: it is rolled up separately.
+
+/**
+ * The ActionBarItem is used in the page header to populate the action bar
+ */
+export let ActionBarItem = ({ ...otherProps }) => {
```

Comment:
If this is going to be made a publicly available component (given that it's still part of `package-settings.js`), we should wrap it in `React.forwardRef()` so that the consumer can pass it a `ref` if they want or need.

If this is just an internal component just disregard (would this component need to be removed from the list of components in `package-settings.js` if that is the case?)

Reply:
Good catch